packages/playground/tests/utils/setup.ts

```diff
@@ -0,0 +1,102 @@
+import { __unstableSchemas, AffineSchemas } from '@blocksuite/blocks/models';
+import { EditorContainer } from '@blocksuite/editor';
+import {
+  type BlobStorage,
+  type DocProviderCreator,
+  type Page,
+  Workspace,
+} from '@blocksuite/store';
+import { createMemoryStorage, Generator, Job, Schema } from '@blocksuite/store';
+
+const createWorkspaceOptions = () => {
```

Comment:
Well we generally prefer the `function createWorkspaceOptions()` style, unless `this` is required inside the function body.